lesson02082021/homework/file/FileWrite.java

```diff
@@ -0,0 +1,56 @@
+package nikitinaalexandra.lesson02082021.homework.file;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Properties;
+
+public class FileWrite {
+    public static final String WORK_DIRECTORY = "src\\main\\java\\nikitinaalexandra\\lesson02082021\\homework\\file";
+    public static final String PROPS_FILE = "data.properties";
+    public static final String SAMPLE_DIR = "newdir";
+
+    public static void main(String[] args) throws IOException {
+        File propsFile = new File(WORK_DIRECTORY + "\\" + PROPS_FILE);
```

Comment:
On Mac I have another separator in path (like /)
To make it working for any OS please use system property file.separator.
You can find an example in attachment in lecture